recipes/vulkan-memory-allocator/all/conanfile.py

```diff
@@ -29,7 +29,7 @@ def layout(self):
         basic_layout(self, src_folder="src")
 
     def requirements(self):
-        self.requires("vulkan-headers/1.3.243.0")
+        self.requires("vulkan-headers/[>=1, <2]")
```

Comment:
Incorrect syntax, it seems: according to [Conan docs](https://docs.conan.io/2/tutorial/versioning/version_ranges.html), there shouldn't be a comma.

The range you specified can also be replaced with a tilde operator:

```suggestion
        self.requires("vulkan-headers/[~1]")
```

Reply:
Yes the comma is needed, I forgot to push the lady commit after I saw a green build 🙈